lib/open-issue.js

```diff
@@ -79,6 +79,7 @@ module.exports = async function (
     newBranch,
     path: 'package.json',
     transform,
```

Comment:
might be nicer to call this in the files we need it rather than passing things into all the jobs?

Reply:
background: I’m trying to get away from more tangled code like this, so that everything that happens in a file is kinda together